.travis.yml

```diff
@@ -2,31 +2,85 @@ language: python
 sudo: false
 
 python:
-  - "2.7"
+- 2.7
 
 env:
-  - TOXENV=flake8
-  - TOXENV=docs
-  - TOXENV=assets
-  - TOXENV=es
-  - TOXENV=addons
-  - TOXENV=devhub
-  - TOXENV=editors
-  - TOXENV=amo
-  - TOXENV=users
-  - TOXENV=main
+  global:
+    - DISPLAY=:99.0
 
 matrix:
   include:
     - python: 2.7
+      env: TOXENV=flake8
+      addons:
+        apt:
+          packages:
+            - swig
```

Comment:
I am not introducing it, it was required before. I am pretty sure it is needed, as I did remove it and got an error when running the app.